### PR TITLE
Fixed code to support Xcode 8.0 beta 4 (8S188o)

### PR DESCRIPTION
**Pull request by learnwithgabbar**
I have fixed the code for xcode 8.0 beta 4

Related to #263

### CODE REVIEW

File: IBAnimatable/AnimatableViewController.swift [hunk 1]
```diff
@@ -30,9 +30,10 @@ import UIKit
   public override func viewWillDisappear(_ animated: Bool) {
     super.viewWillDisappear(animated)
     resetHideNavigationBar()
+
   }
   
-  public override func preferredStatusBarStyle() -> UIStatusBarStyle {
+  public override var  preferredStatusBarStyle:UIStatusBarStyle{
```
Comment (JakeLin) [inline IBAnimatable/AnimatableViewController.swift]: there is a small spacing issue. `public override var preferredStatusBarStyle: UIStatusBarStyle {`

File: IBAnimatable/ExplodeAnimator.swift [hunk 2]
```diff
@@ -73,7 +73,7 @@ private extension ExplodeAnimator {
     for x in stride(from: 0.0, to: Double(size.width), by: Double(size.width / xFactor)) {
       for y in stride(from: 0.0, to: Double(size.height), by: Double(size.width / yFactor)) {
         let snapshotRegion = CGRect(x: CGFloat(x), y: CGFloat(y), width: size.width / xFactor, height: size.height / yFactor)
-        let snapshot = fromViewSnapshot?.resizableSnapshotView(from: snapshotRegion, afterScreenUpdates: false, withCapInsets: UIEdgeInsetsZero)
+        let snapshot = fromViewSnapshot?.resizableSnapshotView(from: snapshotRegion, afterScreenUpdates: false, withCapInsets: UIEdgeInsets.zero)
```
Comment (JakeLin) [inline IBAnimatable/ExplodeAnimator.swift]: I think we can remove `UIEdgeInsets` and use `.zero` only.

File: IBAnimatableApp.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -689,6 +689,7 @@
 					};
 					AE6C66FE1BFBF93500134A35 = {
 						CreatedOnToolsVersion = 7.1.1;
+						DevelopmentTeam = DWK7M63YMR;
```
Comment (JakeLin) [inline IBAnimatableApp.xcodeproj/project.pbxproj]: Can you remove this? We should not commit this info.

File: IBAnimatableApp.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -1055,9 +1056,10 @@
 				ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES = YES;
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
+				DEVELOPMENT_TEAM = DWK7M63YMR;
 				INFOPLIST_FILE = IBAnimatableApp/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks";
-				PRODUCT_BUNDLE_IDENTIFIER = "$(BUNDLE_IDENTIFIER_PREFIX)IBAnimatable.IBAnimatable";
+				PRODUCT_BUNDLE_IDENTIFIER = com.debut.Spotales;
```
Comment (JakeLin) [inline IBAnimatableApp.xcodeproj/project.pbxproj]: Revert this please.

File: IBAnimatableApp/Base.lproj/Main.storyboard [hunk 5]
```diff
@@ -239,6 +239,10 @@
                                                 <real key="value" value="0.59999999999999998"/>
                                             </userDefinedRuntimeAttribute>
                                             <userDefinedRuntimeAttribute type="image" keyPath="leftImage" value="username"/>
+                                            <userDefinedRuntimeAttribute type="string" keyPath="_animationType" value="fade"/>
```
Comment (JakeLin) [inline IBAnimatableApp/Base.lproj/Main.storyboard:242]: I think you can revert this file since all are using for testing?

**Comment by JakeLin** [comment]
@techmehra thank you very much for your PR. I have made some small comments, could you please have a look?

**Comment by learnwithgabbar** [comment]
@JakeLin yes

**Comment by JakeLin** [comment]
@techmehra thanks for your PR. Please let me know if you would like to join and help us with some tasks for upgrading to Swift 3.

**Comment by learnwithgabbar** [comment]
Yes

**Comment by JakeLin** [comment]
@techmehra would you like to join the Slack channel, please send me you email address, you can find mine in github.com/JakeLin

**Comment by Clcanny** [comment]
I meet the same problem in another framework, can you tell me how to fix it by myself and why?

**Pull request merged**